src/mpyl/build.py

```diff
@@ -205,14 +206,20 @@ def find_build_set(
             project_paths,
         )
     )
+    if selected_projects:
+        projects_list = selected_projects.split(",")
```

Comment:
Don't forget the strip any whitespace as well

Reply:
But you can't pass whitespaces via the cli?

Reply:
does it work with -p 'job, ocpp,kong-sync' as well?